data/domain/cards.tsx

```diff
@@ -57,14 +57,18 @@ export class Card {
     }
 
     getStars = (): number => {
-        switch (true) {
-            case this.fivestar && this.count >= Math.floor(this.getCardsForStar(5)): return 5;
-            case this.count >= Math.floor(this.getCardsForStar(4)): return 4;
-            case this.count >= Math.floor(this.getCardsForStar(3)): return 3;
-            case this.count >= Math.floor(this.getCardsForStar(2)): return 2;
-            case this.count >= Math.floor(this.getCardsForStar(1)): return 1;
-            default: return 0;
+        for (let i = Card.getMaximumCardStar(); i > 0; i--) {
```

Comment:
To be perfectly honest, I'm not a fan of this change as I think it makes reading the code much harder. I have to look at 3 different functions to check what the for is doing.

Sorry to make you revert stuff, but do you mind going back to the `switch` code or give me a strong enough reason why this easier to maintain then simply adding another switch case later if a 6th tier comes up?

Reply:
Yeah I think I made it less readable and over complicated it for the sake of being able to easily handle high possibles levels while it'll surely never reach past lv.10 or evern 15 x)

Gonna reverted those changes for both Cards and cardSets